src/index.js

```diff
@@ -519,6 +525,7 @@ export function stream(source: string, index: number=0): () => SourceLocation {
         case DSTRING_START:
         case TSSTRING_START:
         case TDSTRING_START:
+        case HEREGEXP_START:
           setType(STRING_CONTENT);
```

Comment:
Agreed that "string" is a little unintuitive here, but from a lexing/parsing standpoint, I think heregexes can be seen as mostly just strings with a different delimiter and different whitespace trimming rules.